### PR TITLE
WIP Salad UI polishing and Vue container/component organization

### PR DESCRIPTION
**Pull request by lauracw100**
Some re-factoring, component organization and lots of vue styling.

@levackt there are some file naming changes and I combined SaladMix and SaladMixContent.

And then lots of component and scss changes to get the UI close to the figma specs that Ainsley gave us.

### CODE REVIEW

File: src/dapps/Salad/containers/DepositForm/DepositForm.scss [hunk 1]
```diff
@@ -1,93 +1,96 @@
...
-        border: 2px solid #05C0A5;
-        height: 52px;
-        padding-left: 20px;
-        padding-right: 20px;
-        width: 100%;
-
-        @media all and (max-width: $tablet-width) {
-          border-radius: 4px;
+    #deposit-info {
+        position: absolute;
```
Comment (levackt) [inline src/dapps/Salad/containers/DepositForm/DepositForm.scss]: Any reason it's absolute? This combined with the left/top puts it out of screen and unresponsive to screen changes.  Will propose a fix in my branch.

File: src/dapps/Salad/Salad.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+#salad-dapp-container {
+    border: 2px solid #7073b5;
+    background-color: #d0ece9;
+    box-sizing: border-box;
+    width: 715px;
+    height: auto;
+    left: 0px;
+    color: #28464F;
+    @extend %shared-text;
```
Comment (levackt) [inline src/dapps/Salad/Salad.scss:23]: Nice!
Just opened another [little PR](https://github.com/enigmampc/MyEtherWallet/pull/7) for more of this in a global file.

Then we can standardize on primary/secondary buttons, errors etc in there too.

File: src/dapps/Salad/containers/DepositForm/DepositForm.scss (hunk 1 above)
Comment (lauracw100) [reply src/dapps/Salad/containers/DepositForm/DepositForm.scss]: Ah okay. No, I was going by the Figma specs and a newbie at this styling ;-).

File: src/dapps/Salad/Salad.scss (hunk 2 above)
Comment (lauracw100) [reply src/dapps/Salad/Salad.scss:23]: yeah, sounds great.

**Pull request merged**